docs/configuring-kubelinter.md

```diff
@@ -48,6 +48,17 @@ checks:
   addAllBuiltIn: true
 ```
 
+## Ignore paths
+
+To ignore checks on files or directories under certain paths, add ignored paths to `ignorePaths`.
+```yaml
+checks:
+  ignorePaths:
+    - ~/foo/bar
+    - ../baz
+    - /tmp/*.yaml
+```
+
 > Equivalent CLI flag is `--add-all-built-in`
```

Comment:
This one should be moved below the `addBuiltIn` section + it would be nice to add the same for the ignore paths section, i.e.:
```
> Equivalent CLI flag is `--ignore-paths`
```